### DISCUSSION TITLE
Push messages to chats

### DISCUSSION THREAD

**Discussion by KoszticzaM** [open-webui/open-webui#7856] [Q&A]
Hi,
do you guys think I can post messages from a service into  openwebui chats in a notification style? the idea would be the user to get subscribed to some notification feed which pushes messages to her chat, then she is able to ask questions about that specific notification from the assistant. 
my plan so far is to authenticate an admin user through  /api/v1/auths/signin then use /api/v1/chats/{id} to update the chat for each user, who is subscribed.
I look around, but haven't seen anyone doing anything like this so far, do you think this will work? 
one  issue with this is once we update the messages of the chat the ui won't render the changes immediately, so the user won't see the message, is there a way to trigger this?

**Comment by tiangao88** [comment]
I have the same need. Any hints from anyone?

**Comment by lgallindo** [comment]
Bump.
